system-parachains/coretime/coretime-kusama/src/coretime.rs

```diff
@@ -184,9 +227,22 @@ impl CoretimeInterface for CoretimeAllocator {
 		}
 	}
 
-	#[cfg(feature = "runtime-benchmarks")]
-	fn ensure_notify_revenue_info(when: RCBlockNumberOf<Self>, revenue: Self::Balance) {
-		CoretimeRevenue::set(&Some((when, revenue)));
+	fn on_new_timeslice(_t: pallet_broker::Timeslice) {
+		let stash = CoretimeBurnAccount::get();
```

Comment:
Every timeslice (8 mins) still seems quite often to be sending an XCM just to burn accumulated revenue - my initial thought was to do this when a sale is rotated. Every N timeslices might be better, even if just once per day. WDYT?

```suggestion
	fn on_new_timeslice(t: pallet_broker::Timeslice) {
		// Burn roughly once per day. Unchecked math: RHS hardcoded as non-zero.
		if t % 180 != 0 {
			return
		}
```